modules/cli/src/test/kotlin/datamaintain/cli/BaseCliTest.kt

```diff
@@ -14,6 +15,6 @@ open class BaseCliTest {
     }
 
     protected fun runApp(argv: List<String>) {
-        App().subcommands(UpdateDb(runner = ::runner), ListExecutedScripts()).main(argv)
+        App().subcommands(UpdateDb(runner = ::runner), ListExecutedScripts(), MarkOneScriptAsExecuted(runner = ::runner)).main(argv)
```

Comment:
I find it is not safe to need to build the entire CLI in the tests. I would prefer to expose a dedicated method on App that build the CLI as expected.

Reply:
I did some refacto to build CLI with only the subcommand that is going to be tested: f2b67b1348ff6929b5370e69fdc10b5c85276f8f